### DISCUSSION TITLE
Understainding NAMESUBLANG

### DISCUSSION THREAD

**Discussion by Healzangels** [McCloudS/subgen#102] [Q&A] [answered]
Greetings! 
Thanks for creating and sharing such an awesome tool.
One question/sorta issue I was having that maybe some light could be shed on is the NAMESUBLANG.

Right now I have that set to the default in my .env
`NAMESUBLANG='aa'`
However the subtitles generated and pulled into Bazarr only seem to ever show .en.srt

From my understanding I would think that somewhere in that name would be .aa. to make it so in the list of subtitles in say plex for example the whisper generated sub would be at the top of the list and easy to know it was the AI generated. 

I've give it a shot without the '' surrounding the aa but seem to have the same results. Thanks!

**Comment by McCloudS** [comment, marked as answer]
Bazarr will always rename them to en.  NAMESUBLANG is a setting for using the non-Bazarr webhooks and then it’ll look like below with AA:  
![image](https://github.com/user-attachments/assets/7fa1ddfe-4188-4b97-adf9-ea93b64b32d0)

**Reply by Healzangels** [reply]
Gotcha! Makes sense, and thanks for the info!

Had another quick unrelated question, apologies about asking it here vs starting a new topic! 

I've setup Subgen on a 2nd machine to take advantage of it's CPU. Connecting it to Bazarr wasn't an issue but I've found I need to set my Connection/response timeout in seconds very high vs when I had an instance on the same machine that was running Bazarr.

Not sure if this is a misconfiguration on my part or if there really is any "issue" in setting that Connection/response timeout in seconds higher. Currently I've set to the same 3600 as the Transcription/translation timeout in seconds and that has stopped any ConnectionTimeouts.

Thanks again for the info and awesome container. Cheers!

**Reply by McCloudS** [reply]
No issue with the timeout that high as long as you’re getting the results you want.  You may need to set it even higher for longer files on CPU only.  

It essentially will have Bazarr wait up to your 3600 seconds for a subtitle result before marking it as a failure.

**Reply by Healzangels** [reply]
Super, thanks for the additional info!
